junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvSource.java

```diff
@@ -55,4 +55,11 @@
 	 */
 	char delimiter() default ',';
 
+	/**
+	 * The empty value definition to use when reading the {@linkplain #value lines}.
```

Comment:
Please add a second paragraph explaining when the empty value is used.

Reply:
Done